docs/platforms/php/index.mdx

```diff
@@ -25,28 +25,73 @@ Sentry captures data by using an SDK within your application’s runtime. These
 
 Install the SDK using [Composer](https://getcomposer.org/).
 
+
+<OnboardingOptionsButtons
```

Comment:
Would avoid duplicate plurals in the component name:
`OnboardingOptionsButtons` -> `OnboardingOptionButtons`